frontend/tests/Utils.php

```diff
@@ -456,8 +457,7 @@ public static function runGenerateContestCertificates(): void {
              ' --user ' . escapeshellarg(OMEGAUP_DB_USER) .
              ' --database ' . escapeshellarg(OMEGAUP_DB_NAME) .
              ' --password ' . escapeshellarg(OMEGAUP_DB_PASS) .
-             ' --api-token ' . '92d8c5a0eceef3c05f4149fc04b62bb2cd50d9c6' .
-             ' --url ' . escapeshellarg(OMEGAUP_URL))
+             ' --test ' . 'True')
```

Comment:
Acá pudiera ser:
```suggestion
             ' --test True')
```
ya que no le estás mandando ninguna variable, no hay necesidad de concatenar